tools/utils/UtilsTests/ProcessRunnerTests/ProcessRunnerTests.cs

```diff
@@ -0,0 +1,286 @@
+﻿//----------------------------------------------------------------------------------------------------------------------
+// <copyright file="ProcessRunnerTests.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+//----------------------------------------------------------------------------------------------------------------------
+
+namespace UtilsTests
+{
+    using System;
+    using System.IO;
+    using System.Threading;
+    using Microsoft.Packaging.Utils.Logger;
+    using Microsoft.Packaging.Utils.ProcessRunner;
+    using WEX.Logging.Interop;
+    using WEX.TestExecution;
+    using WEX.TestExecution.Markup;
+
+    [TestClass]
+    internal class ProcessRunnerTests : TestBase
+    {
+        private static string toolName = "MakeAppx.exe";
+        private static string appxBundle = "TestAppxBundle.appxbundle";
+        private string testDataDirectory;
+        private string testDirectory;
+
+        /// <summary>
+        /// Test Initialize.
+        /// </summary>
+        [TestInitialize]
+        public new void TestInitialize()
+        {
+            Log.Comment("Initializing ProcessRunner test.");
+
+            // Copy the tool from share under the current directory for testing.
+            this.testDirectory = TestContext.Properties["TestDeploymentDir"].ToString();
+            this.testDataDirectory = this.testDirectory + "peetutils\\test-data\\MakeAppx\\";

```

Comment:
probably not ok for open sourcing

Reply:
My preference is collateral lives with the repo.

---
In reply to: [458387769](https://github.com/microsoft/msix-packaging/pull/358#discussion_r458387769) [](ancestors = 458387769)

Reply:
Added the test data to the repository and updated their names to the correct location.